server/src/main/java/org/opensearch/indices/DefaultRemoteStoreSettings.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.indices;
+
+import org.opensearch.common.settings.ClusterSettings;
+import org.opensearch.common.settings.Settings;
+
+/**
+ * Utility to provide a {@link RemoteStoreSettings} instance containing all defaults
+ *
+ * @opensearch.internal
+ */
+public final class DefaultRemoteStoreSettings {
+    private DefaultRemoteStoreSettings() {}
+
+    public static final RemoteStoreSettings INSTANCE = new RemoteStoreSettings(
```

Comment:
Can we think of this as a DEFAULT_INSTANCE static instance of RemoteStoreSettings class itself?

Reply:
I had thought about it but went with separate class due to 2 reasons:
1. The goal of the separate class is to make it explicit. If we have a static instance in `RemoteStoreSettings`, instead of injecting the instance created in `Node`, we may start using the already created instance.
2. The same setup exists for RecoverySettings